src/App.tsx

```diff
@@ -35,6 +35,8 @@ import { useQuery } from '@apollo/client';
 import { CHECK_AUTH } from 'GraphQl/Queries/Queries';
 import Advertisements from 'components/Advertisements/Advertisements';
 import SecuredRouteForUser from 'components/UserPortal/SecuredRouteForUser/SecuredRouteForUser';
+import FundCampaignPledge from 'screens/FundCampaignPledge/FundCampaignPledge';
+import React from 'react';
```

Comment:
Fix this duplicate identifier error. Add the linting test to the eslint file:
- https://github.com/PalisadoesFoundation/talawa-admin/blob/develop/.eslintrc.json